src/xpanse-api/generated/services.gen.ts

```diff
@@ -140,10 +148,18 @@ import type {
     ManageFailedOrderResponse,
     MigrateData,
     MigrateResponse,
+    ModifyCallback1Data,
```

Comment:
something is not fine. It has generated these duplicate types. 
We need to check what is causing this.

Reply:
Fixed